native/acter/api.rsh

```diff
@@ -2342,9 +2342,6 @@ object Space {
     /// the Tasks lists of this Space
     fn task_lists() -> Future<Result<Vec<TaskList>>>;
 
-    /// the Tasks list of this Space
-    fn task_list(key: string) -> Future<Result<TaskList>>;
```

Comment:
any reason for having it removed ???

Reply:
a) wasn't used and isn't meant to be used anymore - objects should not be taken directly from the spaces, only left the others as tests currently use them. but ideally, we'd remove them as well

b) it was easier to remove the function that wasn't used than include it in the refactor ;) .